next.config.js

```diff
@@ -4,7 +4,7 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
 
 const ContentSecurityPolicy = `
   default-src 'self';
-  script-src 'self' 'unsafe-inline' *.googletagmanager.com 'unsafe-eval';
+  script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com *.google-analytics.com;
```

Comment:
we also need to align `next.config.js`, lets work on seperate PR

Reply:
okay, 
right now, is the admin page accessible?